lib/sprockets/sass_importer.rb

```diff
@@ -1,2 +1,9 @@
-# Deprecated: Require sprockets/sass_processor instead
-require 'sprockets/sass_processor'
+
+require 'sprockets/autoload'
+
+module Sprockets
+  class SassImporter < Autoload::Sass::Importers::Filesystem
+    # NOTE: Hack to support sourcemap generation in Sass-3.3
+    def public_url(a, b);'';end
```

Comment:
:+1: Got it. Change the method definition to `def public_url(*)` and I think this is good to go.